src/main.rs

```diff
@@ -283,7 +283,12 @@ fn main() {
         let working_dir = tmp_path.join(format!("{}", i));
         let source_root = source_root.clone();
 
-        let t = thread::spawn(move || {
+        let mut  thread_name = String::from("Consumer ");
+        thread_name.push_str(&i.to_string());
```

Comment:
You can use `format!`, which would make creating the name nicer (no need for a push)

Reply:
Done!